qiskit/circuit/quantumcircuit.py

```diff
@@ -1143,7 +1144,10 @@ def qasm(self, formatted=False, filename=None):
         Raises:
             ImportError: If pygments is not installed and ``formatted`` is
                 ``True``.
+            QasmError: If circuit has free parameters.
         """
+        if self.num_parameters:
```

Comment:
Just to be more explicit, sometimes these kind of abbreviations gave me debugging headaches (e.g. with the blueprintcircuit) 🙂 
```suggestion
        if self.num_parameters > 0:
```

Reply:
ok, thanks